src/blob/index/bptree/core.rs

```diff
@@ -0,0 +1,429 @@
+/// structure of b+-tree index file from the beginning:
+/// 1. Header
+/// 2. Bloom filter
```

Comment:
I left logic of old `simple` index, but I agree, that maybe we should separate it (and also separate moments, where we send it to file (do it somewhen later). With this we'll filter some requests to not presented keys (=> reduce disk operations)), because it doesn't load memory as mush as btree with indices